src/components/ButtonsLanguages.jsx

```diff
@@ -1,13 +1,18 @@
 import style from './ButtonLanguage.module.css';
-import { useCharacteristicsContext } from '../hooks/useCharacteristicsContext';
 
-function ButtonLanguage({ children, parentName }) {
-  const { handleSelect, handleRemove } = useCharacteristicsContext();
+function ButtonLanguage({ children, isClose, handleSelect, handleRemove }) {
```

Comment:
Please try to think how many props this component really needs.

Check my examples I shared before.